sdk/resourcesdeploymentstacks/arm-resourcesdeploymentstacks/test/resourcesdeploymentstacks_operations_test.spec.ts

```diff
@@ -61,76 +61,78 @@ describe("DeploymentStacks test", () => {
-          mode: "denyDelete"
-        },
-        template:{
-        "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
-        "contentVersion": "1.0.0.0",
-        "parameters": {
-          "foo": {
-            "type": "string",
-            "defaultValue": "foo",
-            "metadata": {
-              "description": "description"
-            }
+      resourcename,
+      {
+        properties: {
+          actionOnUnmanage: {
+            resources: "delete"
           },
-          "bar": {
-            "type": "string",
-            "defaultValue": "bar",
-            "metadata": {
-              "description": "description"
-            }
-          }
-        },
-        "functions": [],
-        "variables": {
+          denySettings: {
+            applyToChildScopes: false,
+            excludedActions: ["action"],
+            excludedPrincipals: ["principal"],
+            mode: "denyDelete"
+          },
+          template: {
+            "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+            "contentVersion": "1.0.0.0",
+            "parameters": {
+              "foo": {
+                "type": "string",
+                "defaultValue": "foo",
+                "metadata": {
+                  "description": "description"
+                }
+              },
+              "bar": {
+                "type": "string",
+                "defaultValue": "bar",
+                "metadata": {
+                  "description": "description"
+                }
+              }
+            },
+            "functions": [],
+            "variables": {
 
-        },
-        "resources": [],
-        "outputs": {
-          "foo": {
-            "type": "string",
-            "value": "[parameters('foo')]"
+            },
+            "resources": [],
+            "outputs": {
+              "foo": {
+                "type": "string",
+                "value": "[parameters('foo')]"
+              },
+              "bar": {
+                "type": "string",
+                "value": "[parameters('bar')]"
+              }
+            }
           },
-          "bar": {
-            "type": "string",
-            "value": "[parameters('bar')]"
-          }
-        }
-      },
+        },
         tags: { tagkey: "tagVal" }
       },
-     testPollingOptions);
+      testPollingOptions);
     assert.equal(res.name, resourcename);
   });
 
   it("deploymentStacks get test", async function () {
-      const res = await client.deploymentStacks.getAtResourceGroup(resourceGroup,
-    	resourcename);
-      assert.equal(res.name, resourcename);
+    const res = await client.deploymentStacks.getAtResourceGroup(resourceGroup,
+      resourcename);
+    assert.equal(res.name, resourcename);
   });
 
-  it("deploymentStacks list test", async function () {
+  it.only("deploymentStacks list test", async function () {
```

Comment:
skip other cases as now I have no permission to test these case
![image](https://github.com/Azure/azure-sdk-for-js/assets/98569699/e488c7bc-3d33-4345-915c-345ed384ef84)